docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md

```diff
@@ -48,7 +48,7 @@ If you choose this option, you can change the configuration to match your deploy
 6. Install the recording rules saved at `operations/mimir-mixin-compiled/rules.yaml` in your Prometheus
 7. Install the alerts saved at `operations/mimir-mixin-compiled/alerts.yaml` in your Prometheus
 
-Alternatively, you can compile the mixin in a container. To do so, replace the previous two steps with the following command:
+Alternatively, you can compile the mixin in a container. To do so, replace steps 3 & 4 with the following command:
```

Comment:
The default `BUILD_IN_CONTAINER` is `true` so the step above `make build-mixin` already does it. I think we can simplify instructions here as follows:
- Remove step "3. Install dependencies"
- Keep step "Compile the mixin" running "make build-mixin" which will use the build-image, with all the deps already installed
- Remove this "Alternatively, you can compile ..." sentence and the following command

Reply:
Ah, didn't realise that was the default. That makes sense :+1: